### ISSUE TITLE
BuildX error when using Ubuntu Focal Fossa 20.04 base image

### ISSUE THREAD

**Issue by ptr727** [ptr727/NxWitness#32]
```
#8 14.76 Created symlink /etc/systemd/system/multi-user.target.wants/systemd-resolved.service → /lib/systemd/system/systemd-resolved.service.
#8 14.77 ln: failed to create symbolic link '/etc/resolv.conf': Device or resource busy
#8 14.77 Created symlink /etc/systemd/system/multi-user.target.wants/ondemand.service → /lib/systemd/system/ondemand.service.
#8 14.77 Created symlink /etc/systemd/system/sysinit.target.wants/systemd-pstore.service → /lib/systemd/system/systemd-pstore.service.
#8 14.78 Initializing machine ID from random generator.
#8 15.02 Setting up networkoptix-metavms-mediaserver (4.3.0) ...
#8 15.09 Detected runtime type: docker.
#8 15.09 System has not been booted with systemd as init system (PID 1). Can't operate.
#8 15.09 Failed to connect to bus: Host is down
#8 15.09 System has not been booted with systemd as init system (PID 1). Can't operate.
#8 15.09 Failed to connect to bus: Host is down
#8 15.16 sysctl: setting key "kernel.core_pattern": Read-only file system
#8 15.17 dpkg: error processing package networkoptix-metavms-mediaserver (--configure):
#8 15.17  installed networkoptix-metavms-mediaserver package post-installation script subprocess returned error exit status 255
#8 15.17 Processing triggers for libc-bin (2.31-0ubuntu9.2) ...
#8 15.19 Errors were encountered while processing:
#8 15.19  networkoptix-metavms-mediaserver
#8 15.20 E: Sub-process /usr/bin/dpkg returned an error code (1)
#8 ERROR: executor failed running [/bin/sh -c apt-get update     && apt-get install --no-install-recommends --yes         ./vms_server.deb     && apt-get clean     && apt-get autoremove --purge     && rm -rf /var/lib/apt/lists/*     && rm -rf ./vms_server.deb]: exit code: 100
------
 > [5/7] RUN apt-get update     && apt-get install --no-install-recommends --yes         ./vms_server.deb     && apt-get clean     && apt-get autoremove --purge     && rm -rf /var/lib/apt/lists/*     && rm -rf ./vms_server.deb:
------
Dockerfile:62
--------------------
  61 |     # Install the mediaserver
  62 | >>> RUN apt-get update \
  63 | >>>     && apt-get install --no-install-recommends --yes \
  64 | >>>         ./vms_server.deb \
  65 | >>>     && apt-get clean \
  66 | >>>     && apt-get autoremove --purge \
  67 | >>>     && rm -rf /var/lib/apt/lists/* \
  68 | >>>     && rm -rf ./vms_server.deb
  69 |     
--------------------
error: failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c apt-get update     && apt-get install --no-install-recommends --yes         ./vms_server.deb     && apt-get clean     && apt-get autoremove --purge     && rm -rf /var/lib/apt/lists/*     && rm -rf ./vms_server.deb]: exit code: 100
Error: buildx call failed with: error: failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c apt-get update     && apt-get install --no-install-recommends --yes         ./vms_server.deb     && apt-get clean     && apt-get autoremove --purge     && rm -rf /var/lib/apt/lists/*     && rm -rf ./vms_server.deb]: exit code: 100
```

Error appears to be in NxMeta 4.3 Beta.
Reverting to Bionic / 18.04 resolves the error.
Switch to Focal / 20.04 when upstream error is resolved.

**Comment by ptr727** [comment]
https://support.networkoptix.com/hc/en-us/community/posts/1500000851801-4-3-R1-fails-to-install-on-Ubuntu-Focal-20-04-LTS-docker-environment

Error:
`sysctl: setting key "kernel.core_pattern": Read-only file system`

```
Preparing to unpack ./vms_server.deb ...
D020000: trigproc_activate_packageprocessing pkg=networkoptix-metavms-mediaserver:amd64

mediaserver-eula license could not be presented
try 'dpkg-reconfigure debconf' to select a frontend other than noninteractive

D020000: post_script_tasks - ensure_diversions
D020000: post_script_tasks - trig_incorporate
Unpacking networkoptix-metavms-mediaserver (4.3.0) ...
D002000: infodb foreach file this pkg
Setting up networkoptix-metavms-mediaserver (4.3.0) ...
D020000: trigproc_activate_packageprocessing pkg=networkoptix-metavms-mediaserver:amd64
Detected runtime type: docker.
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
sysctl: setting key "kernel.core_pattern": Read-only file system
dpkg: error processing package networkoptix-metavms-mediaserver (--install):
 installed networkoptix-metavms-mediaserver package post-installation script subprocess returned error exit status 255
D020000: post_script_tasks - ensure_diversions
D020000: post_script_tasks - trig_incorporate
D010000: trigproc_run_deferred
Errors were encountered while processing:
 networkoptix-metavms-mediaserver
```

**Comment by ptr727** [comment]
Fixed by Nx in 4.3 (5.0) R3: 
https://support.networkoptix.com/hc/en-us/community/posts/1500000851801-4-3-R1-fails-to-install-on-Ubuntu-Focal-20-04-LTS-docker-environment/comments/4409828373655#community_comment_4409828373655

Reverting to Focal base images in 6ef6f11adf8251924ee03b4700c39f5a37a497a5
